.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: python
 python:
- - '3.7.0'
```

Comment:
I've seen breaking differences among the 3.7.x minor versions, so I'd like to preserve a run at 3.7.0 to make sure we don't regress. Maybe we do 3.7.0, 3.7, and 3.8?